### PR TITLE
Feature/control zeta

### PR DESCRIPTION
**Pull request by solidajenjo**

### CODE REVIEW

File: Source/ModuleScene.cpp [hunk 1]
```diff
@@ -560,6 +577,98 @@ void ModuleScene::SaveScene(const GameObject& rootGO, const char* scene, const c
...
+		RELEASE(target.front());
+		target.pop_front();
+	}
+	photoEnabled = false;	
+}
+void ModuleScene::RestorePhoto(GameObject* photo)
+{
+	photoTimer = 0.f;
+	root = photo;
+	std::stack<GameObject*> S;
```
Comment (Rubetoman) [inline Source/ModuleScene.cpp]: Instead of S it will be better to use a more descriptive name (Ex: goStack)

File: Source/ModuleScene.cpp [hunk 2]
```diff
@@ -560,6 +577,98 @@ void ModuleScene::SaveScene(const GameObject& rootGO, const char* scene, const c
...
+	S.push(root);
+	while (!S.empty())
+	{
+		GameObject* go = S.top(); S.pop();
+
+		for (Component* c : go->components)
```
Comment (Rubetoman) [inline Source/ModuleScene.cpp]: Same with the variable name (Better "comp" or "component" instead of "c")

Comment (solidajenjo) [reply Source/ModuleScene.cpp]: solved

File: Source/ModuleScene.cpp (hunk 1 above)
Comment (solidajenjo) [reply Source/ModuleScene.cpp]: solved

**Pull request merged**